requirements.txt

```diff
@@ -1,7 +1,11 @@
-h5py
-numpy
-scipy
-ruamel.yaml
-python-dateutil
-six
-requests
+certifi==2017.7.27.1
+chardet==3.0.4
+h5py==2.7.1
+idna==2.6
+numpy==1.13.3
+python-dateutil==2.6.1
+requests==2.18.4
+ruamel.yaml==0.15.34
+scipy==0.19.1
+six==1.11.0
+urllib3==1.22
```

Comment:
Waiting we have a maintainer documentation section in readthedocs, may be worth adding a comment in the requirements.txt itself explaining how to obtain the list of pinned requirements. (Using pip freeze ..)